ipa-core/tests/common/mod.rs

```diff
@@ -250,11 +251,27 @@ pub fn test_ipa_with_config(mode: IpaSecurityModel, https: bool, config: IpaQuer
         .stdin(Stdio::piped());
     command.status().unwrap_status();
 
+    // Encrypt Input
+    let mut command = Command::new(CRYPTO_UTIL_BIN);
+    command
+        .arg("encrypt")
+        .args(["--input-file".as_ref(), inputs_file.as_os_str()])
+        .args(["--output-dir".as_ref(), path.as_os_str()])
+        .args(["--network".into(), dir.path().join("network.toml")])
+        .stdin(Stdio::piped());
+    command.status().unwrap_status();
+
+    let enc1 = dir.path().join("helper1.enc");
+    let enc2 = dir.path().join("helper2.enc");
+    let enc3 = dir.path().join("helper3.enc");
+
     // Run IPA
     let mut command = Command::new(TEST_RC_BIN);
     command
         .args(["--network".into(), dir.path().join("network.toml")])
-        .args(["--input-file".as_ref(), inputs_file.as_os_str()])
+        .args(["--enc-input-file1".as_ref(), enc1.as_os_str()])
```

Comment:
for now, I've just replaced the old code path with plaintext input data with the new encrypted input code path. do we want to test both paths in these integration tests?

Reply:
I'd say yes - it is good to have most of our code executed in integration tests. If we support it, we should run it

Reply:
Agreed that we should test what we support. Is it useful to continue to support this mode though? In the longer run, maybe we want to only support the encrypted input?